example/cfis_simu/job_sp_simu.bash

```diff
@@ -0,0 +1,462 @@
+#!/usr/bin/env bash
```

Comment:
should be in the `scripts/sh` directory.

Can this be merged with `job_sp.bash`?